init.zsh

```diff
@@ -17,6 +17,29 @@ if ! autoload -Uz is-at-least || ! is-at-least "$min_zsh_version"; then
 fi
 unset min_zsh_version
 
+# zprezto convenience updater
+function zprezto-update {
+    local OWD="$(pwd)"
+    cd -- "${ZPREZTODIR}"
```

Comment:
`pushd` and `popd` would probably be better for this. Then we don't need to track `OWD`. Also note that we should use `-q` to ensure the chpwd functions are not run.